src/main/scala/net/snowflake/spark/snowflake/SnowflakeConnectorUtils.scala

```diff
@@ -113,6 +113,22 @@ object SnowflakeConnectorUtils {
       throw ex
     }
   }
+
```

Comment:
Added 2 Utility methods to disable/enable shared connection.
They are added to `SnowflakeConnectorUtils` because enablePushdown() is in it.